dash-pipeline/SAI/templates/saiapi.h.j2

```diff
@@ -219,7 +199,7 @@ typedef enum _sai_{{ table.name }}_attr_t
     /**
      * @brief Rule priority in table
      *
-     * @type sai_uint32_t
+     * @type sai_int32_t
```

Comment:
Why? Priority can't be negative

Reply:
Yes will remove this change. I think I went by the APP_DB schema which had it as INT32
`priority                 = INT32 value  ; priority of the rule, lower the value, higher the priority`